server/ansible/roles/pbench-repo-install/defaults/main.yml

```diff
@@ -1,8 +1,10 @@
 ---
-test_repo_enabled: 0
+fedoraproject_username: ndokos
+pbench_repo_url_prefix: https://copr-be.cloud.fedoraproject.org/results/{{ fedoraproject_username }}
```

Comment:
@ndokos, @portante, can one of you help me out, here?  It seems weird to have an individual developer's ID in this code.  I presume it's because the repo will have been built under Nick's ID, and, therefore, for any of us to access it, we'll need to reference it appropriately.  But, looking at the options, it looks like other projects are doing their builds under a project ID (at least, I assume that that is what the repos with the `@`-IDs are...).  Should we be following a similar approach?

Reply:
COPR does not let us create an org account, so using my ID has been the fallback. That will go away once the primary mode of installation is a container image. Not sure what the @-ID repos you mention are? What options are you looking at?

Reply:
https://copr-be.cloud.fedoraproject.org/results/

Reply:
@webbnh, can you file an issue to investigate the use of project builds in Fedora Project dot Org?